lib/pigeon.dart

```diff
@@ -1382,9 +1382,6 @@ abstract class NativeCallFlutterApi {
   /// Called to open pin authentication screen.
```

Comment:
Can we check if we can rename this to contain generated in the name.

Reply:
it is possible; i will do it after this branch is merged in another pr